geth/rpc/call.go

```diff
@@ -146,7 +146,7 @@ func (c Call) ParseGasPrice() *hexutil.Big {
 }
 
 // ToSendTxArgs converts Call to SendTxArgs.
-func (c Call) ToSendTxArgs() common.SendTxArgs {
+func (c Call) ToSendTxArgs() utils.SendTxArgs {
```

Comment:
Maybe this conversion between `Call` and `SendTxArgs` should stay on `geth` level? We couple `geth/rpc` and `geth/transactions` just because of this one method.